rotkehlchen/globaldb/assets_management.py

```diff
@@ -53,26 +54,31 @@ def import_assets_from_file(
         # for ethereum tokens comparing by identifier. The edge case of a non-ethereum token
         # with same name and symbol will make this fail.
         asset_type = asset_data['asset_type']
-        asset_ref: Union[Optional[list[str]], Optional[AssetData]]
-        if asset_type == AssetType.EVM_TOKEN:
-            asset_ref = globaldb.get_asset_data(
-                identifier=asset_data['identifier'],
-                form_with_incomplete_data=True,
-            )
-        else:
-            asset_ref = globaldb.check_asset_exists(
-                asset_type=asset_type,
-                name=asset_data['name'],
-                symbol=asset_data['symbol'],
-            )
-        if asset_ref is not None:
-            msg_aggregator.add_warning(
-                f'Tried to import existing asset {asset_data["identifier"]} with '
-                f'name {asset_data["name"]}',
-            )
-            continue
-
         try:
+            if asset_type == AssetType.CUSTOM_ASSET:
+                custom_assets_db = DBCustomAssets(db_handler)
+                custom_assets_db.add_custom_asset(asset_data['extra_information'])
+                continue
+
```

Comment:
this try statement is big and contains things that shouldn't be here. I guess that the only reason is to avoid duplication of the error message. I would have break it in two but since we are changing this part of the code maybe it makes sense to split later

Reply:
yes seems like it. We should not be adding stuff in a big try. But if it will change again anyway very soon it's not critical